### ISSUE TITLE
False alarms on coordinate bounds

### ISSUE THREAD

**Issue by sfinkens** [bcdev/xrlint#64]
**Describe the bug**
xrlint raises some false alarams on coordinate bounds.

**To Reproduce**
```python
import xarray as xr
import xrlint.all as xrl


global_attrs = {
    "Conventions": "CF-1.12",
    "title": "My title",
    "references": "DOI:1234",
    "institution": "My institute",
    "source": "My source",
    "comment": "My comment",
    "history": "My history",
}
lat = xr.DataArray(
    [0.5, 1.5, 2.5],
    dims="lat",
    attrs={
        "standard_name": "latitude",
        "long_name": "latitude",
        "units": "degrees_north",
        "bounds": "lat_bounds"
    }
)
lat_bounds = xr.DataArray(
    [[0, 1], [1, 2], [2, 3]],
    dims=("lat", "bounds"),
)
ds = xr.Dataset(
    {"lat_bounds": lat_bounds},
    coords={"lat": lat},
    attrs=global_attrs
)

# Run linter
linter = xrl.new_linter("recommended")
res = linter.validate(ds)
for msg in res.messages:
    print(msg.node_path, msg.message)
```

Output
```
ds Data variable dimension without coordinates: bounds.
ds.data_vars['lat_bounds'].attrs Missing metadata, attributes are empty.
ds.data_vars['lat_bounds'] Missing attribute 'standard_name'.
ds.data_vars['lat_bounds'] Missing attribute 'long_name'.
ds.data_vars['lat_bounds'] Missing attribute 'units'.
```

**Expected behavior**
No errors, because bounds share their attributes with the corresponding coordinate variable.

**Python Environment**
 - operating system: Rock Linux
 - XRLint version, output of `xrlint --version`: 0.5.1
 - optional: packages and their versions, output of `pip list` or `conda list`: 

**Additional context**
None

**Traceback**
None

**Comment by AliceBalfanz** [comment]
Hi @sfinkens, thanks for reporting! We will have a closer look next week and come back to you :)

**Comment by forman** [comment]
@sfinkens , thanks for pointing out. I agree, coordinates that are bounds for another coordinate do not necessarily need to duplicate attribute values. Let's see if this can be fixed without effort. If so you find the fix in the next version.

**Comment by sfinkens** [comment]
Great, thanks a lot!
